### PR TITLE
Issue/1100

### PR DESCRIPTION
**Pull request by brian-learningpool**

### CODE REVIEW

**Comment by dennisheaney** [comment]
+:cake:

File: plugins/content/course/model.schema [hunk 1]
```diff
@@ -42,6 +42,79 @@
...
+          "inputType": {
+            "type": "Boolean",
+            "options": [
+              true,
+              false
+            ]
+          },
+          "validators": [],
+          "title": "Force routing",
+          "help": "When this is set to 'true', the routing will be forced to the apporpriate start page, regardless of the URL"
```
Comment (ryangadams) [inline plugins/content/course/model.schema]: typo here - in “appropriate”

**Comment by ryangadams** [comment]
+1

**Comment by taylortom** [comment]
+1

**Pull request merged**